float64 thrust
float64 x_torque
float64 y_torque
float64 z_torque
